### DISCUSSION TITLE
Is there a way to add multiple accounts to gmail?

### DISCUSSION THREAD

**Discussion by ekurer** [ferdium/ferdium-app#1542] [Q&A] [answered]
`Add account` leads to sign in the browser and doesn't actually add the account.
<img width="285" alt="297034994-7542edb7-3ae1-4145-9ee1-7cfb96c8aaac" src="https://github.com/ferdium/ferdium-app/assets/60178602/bf17fb60-5e1a-4df9-9ff4-66e0ae036eff">

**Comment by vraravam** [comment, marked as answer]
you need to create a new service by adding the same gmail service.
